### PR TITLE
Improve Filter chip empty state

### PR DESCRIPTION
**Pull request by charlesBochet**
Cases:

image

image

image

### CODE REVIEW

File: packages/twenty-front/src/modules/object-record/object-filter-dropdown/utils/__tests__/isFilterOperandExpectingValue.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    { operand: ViewFilterOperand.IsBefore, expectedResult: true },
+    { operand: ViewFilterOperand.IsAfter, expectedResult: true },
+
+    { operand: ViewFilterOperand.IsNotNull, expectedResult: false },
+    { operand: ViewFilterOperand.IsEmpty, expectedResult: false },
+    { operand: ViewFilterOperand.IsNotEmpty, expectedResult: false },
+    { operand: ViewFilterOperand.IsInPast, expectedResult: false },
+    { operand: ViewFilterOperand.IsInFuture, expectedResult: false },
+    { operand: ViewFilterOperand.IsToday, expectedResult: false },
+  ];
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-front/src/modules/object-record/object-filter-dropdown/utils/__tests__/isFilterOperandExpectingValue.test.ts:23]: good point, removing the edge case from the signature itself

File: packages/twenty-front/src/modules/views/components/EditableFilterChip.tsx [hunk 2]
```diff
@@ -43,7 +44,11 @@ export const EditableFilterChip = ({
     ? `${recordFilter.label} / ${subFieldLabel}`
     : recordFilter.label;
 
-  const labelKey = `${fieldNameLabel}${isNonEmptyString(recordFilter.value) ? operandLabelShort : ''}`;
+  const shouldDisplayOperandLabelShort =
+    isNonEmptyString(recordFilter.value) ||
```
Comment (Weiko) [inline packages/twenty-front/src/modules/views/components/EditableFilterChip.tsx:48]: Should we care about this if we don't expect a value?

**Review by Weiko — APPROVED** [review]
LGTM

File: packages/twenty-front/src/modules/views/components/EditableFilterChip.tsx (hunk 2 above)
Comment (charlesBochet) [reply packages/twenty-front/src/modules/views/components/EditableFilterChip.tsx:48]: yes! the OperandLabelShort should be displayed if we don't exepect a value (: NotEmpty, : NotNull for instance) OR if we have a value (: my value, > my value for instance)
